src/main/java/com/thesamet/gerrit/plugins/saml/SamlWebFilter.java

```diff
@@ -235,9 +235,9 @@ private String getUserName(SAML2Profile user) {
         private AuthenticatedUser user;
 
         public AuthenticatedHttpRequest(HttpServletRequest request,
-                                        @Nonnull AuthenticatedUser user) {
```

Comment:
Actually, it isn't clear to me why the Nonnull annotation was there in the first place. If the class was injected, then it makes sense, otherwise, not sure how it worked.